metrics/density/docker_memory_usage.sh

```diff
@@ -87,6 +91,23 @@ get_runc_pss_memory(){
         echo "$avg"
 }
 
+get_runc_individual_memory() {
+	runc_process_result=$(cat $MEM_TMP_FILE | tr "\n" " " |   sed -e 's/\s$//g')
```

Comment:
can we validate if what we get here is number?

Reply:
I am not clear here do we expect a numerical number or multiples numerical numbers?

Reply:
@jcvenegas it can be a number or multiple numbers it depends of how manny containers are you running